test/npb/ft/npadmana/DistributedFFT.chpl

```diff
@@ -381,7 +373,8 @@ prototype module DistributedFFT {
 
     proc init(type arrType, param ftType : FFTtype, dom : domain(2), parDim : int, signOrKind, in flags : c_uint) {
       this.ftType = ftType;
-      this.parRange = dom.dim(parDim);
+      const (dim1, dim2) = dom.dims();
+      this.parRange = if parDim == 2 then dim2 else dim1;
```

Comment:
Will this `parDim` value eventually want to be 0 or 1 rather than 1 or 2 for consistency with things like `.dim()`?

Reply:
Possibly, I'm not as familiar with this particular section of code. I went with something that works for now so this isn't holding up your PR, and figured we can always make improvements later.